### ISSUE TITLE
[Bug]: Azur Lane Keyboard and Mouse Input Failure  （Sequoia 15.0.1 ）

### ISSUE THREAD

**Issue by Jacob0502** [PlayCover/PlayCover#1777]
### Describe the bug

In Azur Lane, after setting up "key mapping," if the "key mapping" mode is not entered, both keyboard input and mouse clicks will work normally in the game. However, after using the preset hotkeys for a few minutes, both keyboard and mouse click functionality suddenly fail. At this point, the "Option" key becomes unresponsive, and it is not possible to enter "key mapping" mode using hotkeys, although it can still be accessed through the "key mapping" tab at the top. The left mouse button remains functional, but the click feedback only appears in the center of the game screen.

Furthermore, after this issue occurs, quitting and restarting the game results in failure on the second launch, with error code -10670 displayed, and the game only starts correctly on the third attempt.

### Steps to reproduce

I have a stable method to trigger this issue. 
First, I set up a "key mapping" for the game, and then make sure I don't enter "key mapping" mode. At this point, the mouse pointer is visible and functional, and the keyboard keys are also usable. 
Then, I begin using the pre-configured keys on the keyboard. Typically, within two or three minutes, the keys will stop working, and at this point, mouse clicks also stop responding. (To quickly trigger this, simply use the mouse click together with the keys, and it usually takes less than a minute to stabilize into this state.) At this point, the "option" key is unavailable, and I cannot enter "key mapping" mode using the "option" key , but I can still enter key mapping mode through the "key mapping" tab at the top. The pre-configured keys are still not functional, but the left mouse button works, and the UI feedback from clicking only appears in the center of the game screen.
This issue sometimes occurs when "key mapping" is not enabled. 
I usually restart the game to resolve the problem. However, after quitting the game, it always fails to start properly on the second launch (Findar shows error code -10670), and only the third launch works correctly.

<img width="472" alt="截屏2024-12-19 17 01 57" src="https://github.com/user-attachments/assets/1a1de57f-abb5-43a9-9493-adad153746e7" />

<img width="465" alt="截屏2024-12-19 17 03 09" src="https://github.com/user-attachments/assets/70e79167-5542-43dc-8134-19c4711b65c6" />

<img width="271" alt="截屏2024-12-19 17 03 33" src="https://github.com/user-attachments/assets/a7267918-ab4d-44db-9d4d-3a4d7f5fb360" />

### Expected behaviour

I hope to be able to use "key mapping" normally while playing the game, and avoid situations where the keyboard and mouse do not respond.

### What version of PlayCover are you using?

3.1.0

### What version of macOS are you using?

Other (please specify)

### Issue Language

- [X] Yes my issue is written in English

**Comment by Tsukihana1412** [comment]
My computer configuration and app version are the same as yours. I haven't set up keyboard mapping, but from time to time my mouse clicks don't respond. I want to know how to solve this problem.

**Comment by denmanmax** [comment]
Since this has been open for nearly 2 months without any updates, I will mention that I am having a similar issue with Azur Lane keymaps, but without the error code.

I can set up and use the keymaps for maybe a minute, at which point the keymaps stop responding and the cursor gets stuck to the center of the app, sometimes responding to clicks, and also gets stuck 'locked' into the app (option does nothing). Using ⌘+q to close Azur Lane and reopening solves it until I use any of the mapped keys.

I do not have this issue with any other apps.

<i>Apple M3 MacBook Air
macOS Version 15.3 (24D60)
PlayCover Version 3.1.0 (856)</i>

**Comment by Jacob0502** [comment]
> Since this has been open for nearly 2 months without any updates, I will mention that I am having a similar issue with Azur Lane keymaps, but without the error code.
> 
> I can set up and use the keymaps for maybe a minute, at which point the keymaps stop responding and the cursor gets stuck to the center of the app, sometimes responding to clicks, and also gets stuck 'locked' into the app (option does nothing). Using ⌘+q to close Azur Lane and reopening solves it until I use any of the mapped keys.
> 
> I do not have this issue with any other apps.
> 
> Apple M3 MacBook Air macOS Version 15.3 (24D60) PlayCover Version 3.1.0 (856)


Indeed, as you said, it’s been a while since I updated this post. The main reason is that there are now more similar emulators available for Mac, and I’ve moved Azur Lane to other emulators. However, I have to admit that PlayCover remains my favorite emulator.

**Comment by Jacob0502** [comment]
> My computer configuration and app version are the same as yours. I haven't set up keyboard mapping, but from time to time my mouse clicks don't respond. I want to know how to solve this problem.

I can’t fix this issue directly in PlayCover. Here are a few workarounds for you:

1.Set up a dual-boot system on your Mac (install Windows) and use the Mumu emulator (free on Windows).
2.Try other emulators like Mumu (free for the first seven days, then paid).
"Apple adaptive compatibility" (Only Apple can... 😂) <— Just kidding.
